src/osx/cocoa/colour.mm

```diff
@@ -63,57 +127,65 @@ static void Draw(void *info, CGContextRef ctx)
                         CGImageRef image = (CGImageRef) info;
                         CGContextDrawImage
                         (
-                            ctx,
-                            CGRectMake(0, 0, CGImageGetWidth(image), CGImageGetHeight(image)),
-                            image
-                        );
+                         ctx,
+                         CGRectMake(0, 0, CGImageGetWidth(image), CGImageGetHeight(image)),
+                         image
+                         );
                     }
-
+                    
```

Comment:
Could you please get rid of the extra added whitespace? It's trivial, but annoying when reviewing/looking at the diffs later.